src/core/public/application/ui/app_not_found_screen.tsx

```diff
@@ -32,6 +32,10 @@ import { EuiEmptyPrompt, EuiPage, EuiPageBody, EuiPageContent } from '@elastic/e
 import React from 'react';
 import { FormattedMessage } from '@osd/i18n/react';
 
+// A workaround solution
+// it should ideally be replaced by a prop `basePath` passed from the parent component for re-direct
+const baseUrl = window.location.href.split('/app')[0] || '';
```

Comment:
Can you address this workaround?

Reply:
@DarshitChanpura thanks I see what this is serving. However is basePath is set to be something like `/example/app` does this get messy and cause some loop?

Reply:
Actually attempted to add a Prop to this component, but when the route doesn't exist the mounter is undefined so it will be a lot more logic to add get the basePath without any context provided from the mounter.